src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -8,15 +8,20 @@
 #
 FROM registry.access.redhat.com/ubi9/ubi:9.2
 
-ARG ARCH
-ARG PACKER_VERSION
-ARG GO_VERSION
-ARG PROTOC_VERSION
-ARG RUST_VERSION
+ARG ARCH="amd64"
```

Comment:
Making the Dockerfilessimilar to Fedora and Ubuntu will be much desirable. Otherwise it's a pain to figure out what went wrong when debugging issues.